src/main/java/io/vertx/ext/mongo/CreateCollectionOptions.java

```diff
@@ -15,6 +15,7 @@
 public class CreateCollectionOptions {
   private Long maxDocuments;
   private Boolean capped;
+  private TimeSeriesOptions timeseries;
```

Comment:
Please rename this to `timeseriesOptions` for consistency (and then rebuild before committing to update converters).